Sage/carbon
# feat(accordion): describe accordion using design tokens - FE-4501

### Proposed behaviour

Describes a card element using the tokens of the design.
Changes the outline border thickness for focus - 3px.
Updates tests when the above changes are made.

### Current behaviour

Accordion component use the theme styled-component property.
Current outline border thickness for focus - 2px.

### Checklist




#### QA


### Additional context

![image](https://user-images.githubusercontent.com/44143630/149350840-ac2df9b9-a47b-4276-9b9f-880317ba02d5.png)

### Testing instructions

The correct appearance can be checked in the component: accordion.
For theme: mint and aegean there should be no differences.
Use devtools to check if accordion use css variables.
